test/tests/functional/pbs_job_array_comment.py

```diff
@@ -83,7 +83,7 @@ def test_job_array_comment(self):
             ATTR_state: 'B',
             ATTR_comment: (MATCH_RE, 'Job Array Began at .*')
         }
-        self.server.expect(JOB, attr, id=jid, attrop=PTL_AND)
+        self.server.expect(JOB, attr, id=jid, attrop=PTL_AND, offset=5)
```

Comment:
I was wrong about this.  I was actually suggesting adding the offset to the log_match(), but there is no offset parameter to log_match().  It isn't needed here.

Reply:
@bhroam , removed offset.

Reply:
@latha-subramanian , I still see offset being set for expect. Do you mean to remove it?